app/client/src/components/Community.jsx

```diff
@@ -55,6 +56,7 @@ const Community = (props) => {
       >
         <CardHeader
           onClick={handleCommunityClick}
+          style={{ cursor: "pointer" }}
```

Comment:
We need to be using the 'sx' prop instead of the 'style' prop. the 'sx' prop is theme-aware so it will be important for dark mode integration and maintaining application UI consistency.